nilearn/plotting/displays.py

```diff
@@ -88,6 +88,9 @@ def draw_2d(self, data_2d, data_bounds, bounding_box,
             raise ValueError('Invalid value for direction %s' %
                              self.direction)
         ax = self.ax
+        # Switches into contour fillings by overriding type as 'contourf'
+        if 'fill' in kwargs and kwargs['fill'] is True:
+            type = 'contourf'
```

Comment:
If I add ```type='contourf'```, will it not disturb other contour plotting examples ?
Did I understood your comment wrongly?